benchmarks/dense_matmul/genbenchmark.py

```diff
@@ -80,23 +80,31 @@ def generate_dense_benchmark(m, n, k) -> SNAXBenchmark:
     return bm
 
 
-def output_log() -> str:
+def output_log(output_report) -> str:
     result = "# Dense Matmul Benchmark Results\n\n"
     dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
     result += f"This test was run at {dt_string}\n\n"
-    result += "| benchmark | cycles | ideal | utilization |\n"
-    result += "| --- | --- | --- | --- |\n"
-    avg_utilization = 0
-    avg_n = 0
-    for benchmark in output_report:
-        result += f"| [{benchmark}]({benchmark}) "
-        result += f"| {output_report[benchmark]['cycles']} "
-        result += f"| {output_report[benchmark]['ideal']} "
-        result += f"| {output_report[benchmark]['utilization']} | \n"
-        avg_utilization += output_report[benchmark]["utilization"]
-        avg_n += 1
-    result += "| average | | |"
-    result += f"{avg_utilization/avg_n} |\n"
+    for layout in ("cyclic", "banked"):
+        result += f"Results for a {layout} layout \n\n"
+        result += "| benchmark | layout | M | N | K | cycles | ideal | utilization |\n"
+        result += "| --- | --- | --- | --- |\n"
```

Comment:
I'd like to deduplicate this test a bit from the memory layout. (E.g. have a test that tests utilization vs matrix size.
And have a separate test that tests utilization vs size vs layout ?) But I'm not sure that makes sense?

Reply:
it seems now we have everything? (utilization vs matrix size vs layout)? In the outputs, the results are separated by layout